packages/atomic/src/components/result-template-components/atomic-result-icon/atomic-result-icon.tsx

```diff
@@ -41,14 +43,25 @@ export class AtomicResultIcon {
     return objectType || fileType || 'custom';
   }
 
-  public render() {
+  private get isPath() {
+    return !!this.icon?.match(/^(https?:\/\/|\.\/|\.\.\/)/);
+  }
+
+  public async componentWillLoad() {
     const icon = this.icon || this.defaultIcon;
-    const iconPath = getAssetPath(`./assets/${icon}.svg`);
+    const iconPath = this.isPath ? icon : getAssetPath(`./assets/${icon}.svg`);
+    const response = await fetch(iconPath);
+    this.svg =
+      response.status === 200 || response.status === 304
+        ? await response.text()
+        : null;
+  }
 
+  public render() {
     return (
       <Host
-        class={icon}
-        style={{'background-image': `url(${iconPath})`}}
+        class={!this.isPath ? this.icon || this.defaultIcon : ''}
+        innerHTML={this.svg}
       ></Host>
```

Comment:
I made these changes for two reason:
1. Supporting custom colors on SVGs.
2. Supporting CDN/relative links to SVGs.

Do you believe we should support those?

Reply:
For the rating facet icons, we do currently do not support CDN/ relative links to SVGs. Perhaps this is something that should be consistent between the two.

Reply:
I agree. I'll do that in a separate PR.